connectivity/FEATURE_BLE/source/cordio/source/GattServerImpl.cpp

```diff
@@ -633,7 +642,7 @@ ble_error_t GattServer::read(
     uint8_t *att_value = nullptr;
 
     if (AttsGetAttr(att_handle, &att_length, &att_value) != ATT_SUCCESS) {
-        tr_error("Cannot read attribute (not able to get length)");
+        tr_error("not able to get attribute length");
```

Comment:
???
```
tr_error("Attribute with handle &d not found.", att_handle);
```

Reply:
Yup. It is failing to get the attribute length because the attribute can't be found. I have remedied this in commit 6216ad31f4a25e7c744e6b82bdc7c56a678ac8db.